sdk/src/types/block/slot/epoch.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use derive_more::{Deref, Display, From, FromStr};
+
+/// Timeline is divided into slots, and each epoch has a corresponding epoch index.
```

Comment:
```suggestion
/// Timeline is divided into epochs, and each epoch has a corresponding epoch index.
```
Or should it be epochs here ?

Reply:
done